CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 4.3.0
+
+- Apply changes to support the getAsync() interface used in [kartotherian](https://github.com/wikimedia/mediawiki-services-kartotherian)
```

Comment:
Upstream also implements async tile fetch, but after our fork it seems: https://github.com/mapbox/tilelive-vector/commit/bac7fe06a7089073d37fa86c392e2489c0aca35f